src/api/app/models/package.rb

```diff
@@ -839,6 +839,7 @@ def add_channels(mode=nil)
         next if mode == :skip_disabled and not cb.channel_binary_list.channel.is_active?
         cpkg = cb.create_channel_package_into(self.project)
         next unless cpkg
+        next if mode.nil? and not cb.channel_binary_list.channel.is_active?
```

Comment:
This is the second commit, it is fixing exactly what the commit message tells:"it should not add repos of disabled channels"